fix: updateMany use undefined variable; (#34)
RonasIT/laravel-helpers: documentation/traits.md

## FixturesTrait

> Is a powerful tool which make testing process understandable and cleaner. 

Reviewer: ```suggestion
This is a powerful tool which makes testing process understandable and cleaner.
```

### Fixtures

All additional data such as the results of the operation or input data can be
presented via `.json` files and grouped by the test case name in the path
`/tests/fixtures/{$testClassName}`.

> The most common fixtures helpers presented below:

Reviewer: ```suggestion
The main fixture helpers are following:
```

- `getFixture($fn, $failIfNotExists = true)`
- `getJsonFixture($fn, $assoc = true)`
- `assertEqualsFixture($fixture, $data, bool $exportMode = false)`
- `exportJson($fixture, $data)`
- `exportContent($content, $fixture)`
- `exportFile($response, $fixture)`

### DB prefilling

 Trait also provides an ability to use the same initial database state before
each test running. Testing database will be automatically cleared in the `setUp`

> method and restore from the `/tests/fixtures/{$testClassName}/dump.sql` file.

Reviewer: ```suggestion
method and restored from the `/tests/fixtures/{$testClassName}/dump.sql` file.
```
